### ISSUE TITLE
vrdio does not start automatically after initial config in SteamVR home

### ISSUE THREAD

**Issue by bonsi** [Supreeeme/vrdio#1]
First of all: THANK YOU so much for helping me fix audio in SteamVR!

So the problem is that `vrdio` does NOT start automatically after the initial configuration. 

What I did:
- run `./vrdio-launch.sh`   while SteamVR is running
- this made it possible to configure audio via SteamVR
- restart SteamVR
- vrdio does NOT show up anymore, although it's loaded as an overlay in SteamVR settings. No audio at this point in the headset
- if I start `./vrdio-launch.sh` again at this point, vrdio becomes visible and accessible again in SteamVR and audio instantly starts to work again

What I found in the SteamVR output when starting SteamVR from the cmdline:

`/home/me/Downloads/vrdio/vrdio-release-1.0/bin/vrdio: error while loading shared libraries: libQt6Quick.so.6: cannot open shared object file: No such file or directory`

My system:
OS: Ubuntu 20.04 LTS
Audio server: pipewire
VR: Valve Index
Qt version: 5

`lsb_release -a`
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.5 LTS
Release:	20.04
Codename:	focal

`pactl info`
Server String: /run/user/1000/pulse/native
Library Protocol Version: 33
Server Protocol Version: 35
Is Local: yes
Client Index: 3739
Tile Size: 65472
User Name: ivo
Host Name: dubuntu.home.internal
Server Name: PulseAudio (on PipeWire 0.3.60)
Server Version: 15.0.0
Default Sample Specification: float32le 2ch 48000Hz
Default Channel Map: front-left,front-right
Default Sink: alsa_output.pci-0000_00_1f.3.pro-output-0
Default Source: alsa_input.pci-0000_00_1f.3.pro-input-0
Cookie: 83ff:044a

`qmake --version`
```
QMake version 3.1
Using Qt version 5.12.8 in /usr/lib/x86_64-linux-gnu
```
`qtdiag`
`qtdiag: could not find a Qt installation of ''`

Thanks in advance for taking a look at this.
Let me know if you need more info.

**Comment by Supreeeme** [comment]
Hm, guess I never ran into this because I actually have Qt6 installed. Gonna see if I can try and reproduce in a chroot.

**Comment by bonsi** [comment]
As far as I am concerned: don't put too much effort into it because I'm going to have to upgrade to the latest Ubuntu LTS (22.04) soon enough anyway. It won't be an issue anymore then because the default Qt version is 6 as far as I know.

**Comment by Supreeeme** [comment]
Yeah, not having much luck getting it running inside the chroot at all, even with qt6 installed. Maybe I'll just make Qt6 a hard requirement. Could you try this though:
- Copy `~/.config/vrdio/vrdio.vrmanifest` to some other folder
- Uninstall vrdio (`./vrdio-launch.sh --uninstall`) and then shut down SteamVR (important)
- Move `vrdio.vrmanifest` back to the `vrdio` folder
- Open `vrdio.vrmanifest` and change the `binary_path` value to point to `vrdio-launch.sh`
- Launch SteamVR, run vrdio, and then shut down SteamVR

Can you check if on the next SteamVR launch it launches properly?
